### ISSUE TITLE
Issue with Moodle 3.5 and 3.6 ( Not updating current course)

### ISSUE THREAD

**Issue by audetcameron** [moodleuulm/moodle-local_boostnavigation#19]
I was having an issue where the active course would not update, it also affected the breadcrumb navigation (only on course over view pages)  all other pages worked fine.

**Comment by jonlan** [comment]
I observed that the visibility of the breadcrumb (link highlighted or not when visibility of the course is set) depends on whether the Block "My Courses" is collapsed or not collapsed.

**Comment by abias** [comment]
Hi Cameron,

I am afraid that I don't fully understand the issue which you are reporting.
Could you please provide steps to reproduce (which settings did you set, what did you expect to see in the navigation, what did you really see), based on a plain installation of local_boostnavigation on Moodle 3.5 or 3.6? I will then be happy to check your problem here.

Thanks,
Alex

**Comment by audetcameron** [comment]
Hi Alex,
First, let me thank you for your hard work and this amazing addition. It's a huge improvement over the default boost navigation settings. 

I updated Moodle from 3.4 to 3.6.  
I cleared all caches and even did a fresh 3.6 install on a new server.

I was experiencing an issue where the active class in the navigation wouldn't change regardless of what course I had selected.  It also had an effect on the breadcrumb navigation staying the same.  They all registered as the first class as if I were only viewing that course.  I disabled the plugin and it started working again.  I have since re enabled the plugin and it appears to be working again as expected, I will update you if I experience the same issue and try to take video/screenshots to help.

Thanks for reaching out.
-Cameron
